samples/scripts/deployment/localDeployment.bat

```diff
@@ -7,6 +7,7 @@ set HOST_NAME=%1
 set NODE_NAME=%2
 set NODE_SOURCE_NAME=%3
 set RM_URL=%4
+set NB_NODES=%5
```

Comment:
This change could break existing scripts that use _localDeployment.bat_. To prevent that, you could check if $5 is defined, if not then a default value should be set.